File: HomeworkManager/HomeworkManager.Model/CustomEntities/Appointment/AppointmentModel.cs
﻿namespace HomeworkManager.Model.CustomEntities.Appointment;

public class AppointmentModel
{
    public required int AppointmentId { get; set; }
    public required string Time { get; set; }
    public required string TeacherName { get; set; }
    public required string TeacherEmail { get; set; }
    public required bool IsAvailable { get; set; }
    public required bool IsMine { get; set; }
}

File: HomeworkManager.App/app/src/main/java/hu/bme/aut/android/homeworkmanagerapp/domain/model/appointment/AppointmentModel.kt
package hu.bme.aut.android.homeworkmanagerapp.domain.model.appointment

data class AppointmentModel(
    val appointmentId: Int,
    val time: String,
    val teacherName: String,
    val teacherEmail: String,
    val isAvailable: Boolean,
    val isMine: Boolean
)